src/js/stores/__tests__/BaseStore-test.js

```diff
@@ -0,0 +1,65 @@
+jest.dontMock('../BaseStore');
+
+var BaseStore = require('../BaseStore');
+
+describe('BaseStore', function () {
+
+  beforeEach(function () {
+    this.instance = new BaseStore();
+  });
+
+  describe('#get', function () {
+
+    it('should return undefined if no key is given', function () {
+      expect(this.instance.get()).toEqual(null);
+    });
+
+    it('should return undefined if given an object', function () {
+      expect(this.instance.get({})).toEqual(null);
+    });
+
+    it('returns null if property hasn\'t been defined', function () {
+      expect(this.instance.get('foo')).toEqual(null);
```

Comment:
I agree on that and I would like to completely remove the `get` and `set` methods but that's a different story. The intention of this store was to replace the `Store.create` util as well as the `GetSet` mixin, thus it replicates their interface and we should not alter it for now.